### ISSUE TITLE
Service behind the LB and HTTP inside the manifest instead of the HTTPS

### ISSUE THREAD

**Issue by qchroman** [kaltura/nginx-vod-module#1237]
Hello all,

Ask you know in order to test Widevine DRM on content, the content must be delivered from a secure location and the SSL certificate must be valid. In order to make the set up easier, usually, the server that produces the content is set up behind the LB and LB takes care of the SSL certificate.

What I have noticed in this case, is that the dash manifest has the HTTP inside the media/initialization properties for the URL. Is it possible to tell the module to use HTTPS even if someone addresses it over HTTP?

Thank you in advance.

**Comment by qchroman** [comment]
seems that setting the 

`vod_base_url "https://$http_host";`

inside the nginx.conf did the trick. Is it a correct approach?

**Comment by erankor** [comment]
Yes, we are mapping it based on some header returned from the load balancer, but if you don't need http, you can just hardcode it to https.
